cmd/ipfs/daemon.go

```diff
@@ -461,3 +478,80 @@ func merge(cs ...<-chan error) <-chan error {
 	}()
 	return out
 }
+
+func runGC(req cmds.Request, repo repo.Repo, node *core.IpfsNode) error {
+	disableGC, _, err := req.Option(disableGCKwd).Bool()
```

Comment:
This is outside of `daemonFunc` because it is cleaner this way. But it is the other way around for `mountFuse`, `initWithDefaults`. Which one is preferred? With `runGC`'s, there is less `res.SetError(err, cmds.ErrNormal)`.